`$ gatsby develop` fails with webpack error

Hi there, 

First of all, thank you for this starter, it looks great! I'm trying to build it on macOS Catalina with the latest Gatsby CLI / npm installed and I keep getting this error on `gatsby develop`
```
failed Building development bundle - 14.666s

 ERROR #98123  WEBPACK

Generating development JavaScript bundle failed


/Users/username/Documents/Git/project-folder/src/lib/reset.js
  6:17  error  React Hook "useTheme" is called in function "resetStyles" which is neither a React function component or a custom React Hook function  react-hooks/rules-of-hooks

✖ 1 problem (1 error, 0 warnings)


File: src/lib/reset.js
 ```

Is there something I'm doing wrong or is this a known issue?

--- a/src/lib/reset.js
+++ b/src/lib/reset.js
@@ -2,7 +2,7 @@ import { css } from '@emotion/core'
 import { useTheme } from '../components/Theming'
 import typography, { fonts } from '../lib/typography'
 
-const resetStyles = () => {
+const ResetStyles = () => {
   const theme = useTheme()
   return css`
   form {
@@ -157,4 +157,4 @@ const resetStyles = () => {
 `
 }
 
-export default resetStyles
+export default ResetStyles

